iteration/iteration.go

```diff
@@ -127,7 +131,7 @@ func (m *GormIterationRepository) List(ctx context.Context, spaceID uuid.UUID) (
 	return objs, nil
 }
 
-// Get the Root Iteration for a space
+// RootIteration returns the Root Iteration for a space
 func (m *GormIterationRepository) RootIteration(ctx context.Context, spaceID uuid.UUID) (*Iteration, error) {
```

Comment:
Done in https://github.com/almighty/almighty-core/pull/990/commits/d1115510d01710c3882ed1b8845f8ee144f251dd
thanks.